src/sensors/mpu6050sensor.cpp

```diff
@@ -135,6 +137,21 @@ void MPU6050Sensor::motionLoop()
         quaternion.set(-rawQuat.y, rawQuat.x, rawQuat.z, rawQuat.w);
         quaternion *= sensorOffset;
 
+        VectorFloat gravity;
+        imu.dmpGetGravity(&gravity, &rawQuat);
+        // dmpgetGravity returns gravity in 4g mode so double it to get the 2g mode vector
```

Comment:
Are you sure? It should return gravity according to set sensor mode, and we use 2G.

Reply:
I'm not sure, but I did this because the output showed half a G downwards while sitting still without this doubling. Also dropping the sensor shows one G of accel with this doubling

Reply:
That's weird, but okay.